internal/pkg/consensus/validation.go

```diff
@@ -19,6 +19,21 @@ var errInsufficientGasCt *metrics.Int64Counter
 var errNonceTooLowCt *metrics.Int64Counter
 var errNonceTooHighCt *metrics.Int64Counter
 
+var (
+	// These errors are only to be used by ApplyMessage; they shouldn't be
```

Comment:
Since ApplyMessage is now deleted should these also be deleted?

Reply:
I deleted the unused, there are still references to this somewhere. 
Making a pass across the entire codebase to delete all the unsed/legacy code will take some effort